common/src/main/java/com/viaversion/viaversion/protocols/protocol1_12to1_11_1/Protocol1_12To1_11_1.java

```diff
@@ -167,6 +169,10 @@ public void registerMap() {
                     ClientWorld clientChunks = wrapper.user().get(ClientWorld.class);
                     int dimensionId = wrapper.get(Type.INT, 1);
                     clientChunks.setEnvironment(dimensionId);
+
+                    // Reset recipes
+                    wrapper.create(ClientboundPackets1_13.DECLARE_RECIPES, packetWrapper -> packetWrapper.write(Type.VAR_INT, 0))
+                            .scheduleSend(Protocol1_13To1_12_2.class);
```

Comment:
Oh and this should be `Protocol1_12To1_11_1.class`, the default behavior of that method is to skip current pipeline (aka start from the target)